metadata-ingestion/src/datahub/ingestion/source/openapi.py

```diff
@@ -348,14 +359,16 @@ def get_workunits_internal(self) -> Iterable[ApiWorkUnit]:  # noqa: C901
                         fields2add, _ = extract_fields(response, dataset_name)
                         if not fields2add:
                             self.report.report_warning(
-                                key=endpoint_k, reason="No Fields"
+                                title=endpoint_k,
```

Comment:
endpoint_k definitely isn't the title we want

Reply:
this source's errors were a mess